apps/modernization-ui/src/pages/patient/profile/administrative/AdministrativeTable.tsx

```diff
@@ -132,9 +130,55 @@ export const AdministrativeTable = ({ patient }: Props) => {
                 break;
         }
     };
+
+    const generateTableRow = (administrative: PatientAdministrative, index: number): TableBody => {
+        return {
+            id: index,
+            tableDetails: [
+                {
+                    id: 1,
+                    title: administrative?.asOf ? internalizeDate(administrative?.asOf) : null
+                },
+                {
+                    id: 2,
+                    title: administrative?.comment
+                },
+                {
+                    id: 3,
+                    title: (
+                        <div className="table-span">
```

Comment:
here we already have this component **<PatientTableActions>** so it might be preferred in this case?

Reply:
I can't get PatientTableActions to disable delete 
![Screenshot 2023-12-13 at 12 13 19 PM](https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/60e6b029-1ef1-4797-9ebc-d348cd8fe725)

Reply:
`ActionState` has a `notDeletable` property, perhaps this could be added to `PatientTableActions`?